website/docs/reference/project-collaboration-mode.md

```diff
@@ -2,52 +2,64 @@
 title: Project Collaboration Mode
 ---
 
-:::info Availability
+:::note Availability
 
-The project collaboration mode is an enterprise-only feature that was introduced in **Unleash 4.22.0**.
+**Plan**: [Enterprise](https://www.getunleash.io/pricing) | **Version**: `4.22+`
 
 :::
 
-A project's **collaboration mode** specifies **who can submit [change requests](change-requests.md)**. There are two collaboration modes:
-- [**open**](#open-collaboration-mode)
-- [**protected**](#protected-collaboration-mode)
-- [**private**](#private-collaboration-mode)
+# Overview
 
-## **Open** collaboration mode
+Project collaboration modes allow you to manage project visibility and interaction levels, ensuring better control and collaboration within your organization.
 
-Anyone can submit change requests in an **open** project, regardless of their permissions within the project and globally. This is the default collaboration mode for projects.
+The collaboration mode of a [project](projects.md) defines who within your Unleash instance can access the project. This setting also determines who can submit [change requests](change-requests.md)–a feature that provides controlled and secure updates for feature toggles.
```

Comment:
Couple small things:
1. We're working the docs (and everything else) towards "flags" instead of "toggles" (you've probably heard this by now, but just in case you haven't), so let's stick with that going forward.

2. Should we use spaces around hyphens/em-dashes that are used for interjections or not? I think I've always preferred using spaces, especially because people don't tend to differentiate between hyphens and em-dashes, so leaving out the spaces can technically cause some ambiguity (though it's probably not an issue). Honestly no strong feelings on this, just wanna align; your call.

3. I'm not sure about the description of change requests there. Is it necessary? And more importantly: is that the right way to describe them? It kinda sounds like marketing lingo, to me, which feels weird in the docs. But let's talk about it 💁🏼 
 
```suggestion
The collaboration mode of a [project](projects.md) defines who within your Unleash instance can access the project. This setting also determines who can submit [change requests](change-requests.md) – a feature that provides controlled and secure updates for feature flags.
```

Reply:
1. Of course!
2. Understood! I'll try to keep that in mind for the next time I write something with dashes.
3. That's a good point. I think my issue is that "controlled and secure updates for feature flags" sounds like something that belongs on a marketing website to me. It doesn't really tell me anything about the feature. At least it feels like fluff to **me**. But you might not agree, and it's your decision in the end, so it's entirely up to you. I've voiced my opinion, I'm very happy to be overruled 💁🏼